common/src/main/java/com/wynntils/gui/screens/maps/GuildMapScreen.java

```diff
@@ -73,6 +91,22 @@ protected void init() {
                                         "screens.wynntils.guildMap.toggleResourceColor.name")),
                         new TranslatableComponent("screens.wynntils.guildMap.toggleResourceColor.description")
                                 .withStyle(ChatFormatting.GRAY))));
+
+        territoryDefenseFilterButton = this.addRenderableWidget(new BasicTexturedButton(
+                width / 2 - Texture.MAP_BUTTONS_BACKGROUND.width() / 2 + 6 + 20 * 1,
+                (int) (this.renderHeight
+                        - this.renderedBorderYOffset
+                        - Texture.MAP_BUTTONS_BACKGROUND.height() / 2
+                        - 6),
+                16,
+                16,
+                Texture.MAP_ADD_BUTTON, // TODO: Add new cycle texture
+                (b) -> {
+                    territoryDefenseFilterLevel = territoryDefenseFilterLevel.next();
+                    territoryDefenseFilterButton.editTooltip(
+                            getTerritoryDefenseFilterButtonTooltip(territoryDefenseFilterLevel));
+                },
+                getTerritoryDefenseFilterButtonTooltip(territoryDefenseFilterLevel)));
```

Comment:
At some point we should create functionality to automatically add buttons at the right location

Reply:
`width / 2 - Texture.MAP_BUTTONS_BACKGROUND.width() / 2 + 6 + 20 * buttonSlot`, already made it so you don't have to guess.